packages/tools/pie-icons-configs/__tests__/configs.test.js

```diff
@@ -144,23 +146,45 @@ describe('getSvgProps', () => {
 
         describe('when provided with invalid parameters', () => {
             it('returns an object with the expected classes', () => {
-                const received = getSvgProps('pie-icon pie-icon--app-order', 'FOO', null, iconName);
+                const received = getSvgProps('pie-icon pie-icon--app-order', null, 'n', iconName);
```

Comment:
Bit confused about this change. Did the test pass fine with the old params – and if so, should we be testing other outputs as well? Or is it ok that the tests pass when both `…'Foo', null` is passed in and `null, 'n'`, but the expects are the same?

Reply:
Test pass with both old and new params. I just changed it for consistency. 'FOO' parameter was not related to the test, as it is a static css class that can be passed, and 'n' is invalid value while null is a valid value in this case that should mean that default width and height should be applied without a console error (covered in test below). I can rewrite the descriptions to make it clearer.